bemuse/src/game/ui/LoadingScene.jsx

```diff
@@ -46,7 +46,7 @@ export default class LoadingScene extends React.Component {
     }
     this.props.registerTeardownCallback(() => {
       ReactDOM.findDOMNode(this.refs.scene).classList.add('is-exiting')
-      return Promise.delay(500)
+      return new Promise((resolve) => setTimeout(resolve, 500))
```

Comment:
[A `delay` npm package](https://www.npmjs.com/package/delay) exists for this.

Reply:
Fixed on faf9f8eb.